insights/client/apps/compliance/__init__.py

```diff
@@ -154,3 +153,6 @@ def _assert_oscap_rpms_exist(self):
             if len(rpm.strip().split('\n')) < len(REQUIRED_PACKAGES):
                 logger.error('Missing required packages for compliance scanning. Please ensure the following packages are installed: {0}\n'.format(', '.join(REQUIRED_PACKAGES)))
                 exit(constants.sig_kill_bad)
+
+    def _get_inventory_id(self):
+        self.conn._fetch_system_by_machine_id()[0]['id']
```

Comment:
WIP for error handling

Reply:
@gravitypriest (or @SteveHNH) `_fetch_system_by_machine_id` was clearly meant to be private (Python's soft-private leading underscore), but I see other places in the code using this method publically. I just wanted to check with you that this is OK to use.

Reply:
Yep, no worries there. It will return an HTTP response body so if it works for you purposes, by all means. I think this could feasibly be a public method anyway.